build-scripts/deploy-github.js

```diff
@@ -44,8 +44,10 @@ function getReleaseInfo() {
     const tag = process.env.TAG; // eslint-disable-line no-undef
         throw new Error("No TAG environment variable");
     }
-
+    /* eslint "no-warning-comments": [1, { "terms": ["todo", "fixme"] }] */
+    /* eslint-disable*/
     return {
```

Comment:
Hi @samgiles! After enabling eslint I am getting an error in this return:
/home/travis/build/mozilla-magnet/magnet-client/build-scripts/deploy-github.js
  49:5  error  Parsing error: 'return' outside of function
[Travis run](https://travis-ci.org/mozilla-magnet/magnet-client/builds/133430869#L3232)

Not sure how I should fix it, could you please have a look at it? 
I tried to disable that line for eslint temporarily but so far I do not get it.
Thanks!